usr/share/rear/restore/NBKDC/default/40_restore_backup.sh

```diff
@@ -0,0 +1,16 @@
+LogPrint "
+
+The System is now ready for restore. Please start the restore task from the NovaBACKUP DataCenter Central Management!
+
+! Remember that the restore target must be set to '/mnt/local' !
+
+For further documentation see the following link:
+
+ http://www.novastor.com/help-html/dc/en-US/index.html
+
+
+after the restore has finished quit this wizard with the command 'exit' to continue.
+"
+
+rear_shell "Did you restore the backup to /mnt/local ? Are you ready to continue recovery ?"
```

Comment:
We need to wait in the recovery sequence until the data is written, and to then continue with the recovery (boot loader and inird stuff) 
SEP was using the same method to "wait" for user input (the 'exit') until the restore of data completes, to then continue with the recovery process. Thats why I took it from them.
Currently I am on the brink of writing a wait loop from which the user input is read as a trigger to continue. So I wont need the actual rear_shell anymore. :-)